Brandon_Becker.cfdg

```diff
@@ -0,0 +1,12 @@
+startshape sierpinski
+background { hue 0 sat 1 b -1 }
+rule tr{
```

Comment:
The sierpinski triangles are so cool! Very fun, and I hadn't even considered calling a rule on itself for this assignment. Am I right in assuming then that the art stops when the length of a shape would be negative? I also love the colors, the vaguely transparent purple has a nice contrast.